feat(Challenge Page): A page for community challenges

TTalex opened the pull request:
### What
- See #934 for description, and questions
- This PR is only used to see code diffs for the prototype, it is not ready for merging


### Fixes bug(s)
- Fixes #934

teolemon commented:
@TTalex do you have screenshots of this ?

raphodn commented:
> @TTalex do you have screenshots of this ?

There is 1 in the related issue (#934)

TTalex commented:
Some updates with the upcoming Nutella challenge:
* Added two tutorial cards leading to multiple proof uploader and Price validation assistant
* Changed title, dates, example image and tracked categories to match the new challenge
* Moved most text to translations
* Added stats on proof uploads
* Moved page to experiments
* Make API calls work using https://github.com/openfoodfacts/open-prices/pull/675

New Page look:
![Capture1](https://github.com/user-attachments/assets/0c9a43f2-5dfa-4330-8492-f9e2432e213e)

TTalex commented:
Update:
* Removed elements requiring group by stats (Top contributors, user ranking, chart)
* Reverted change on PriceChart, as it is no longer used
* Edit filter name from `product__categories_tags__contains` to `product__categories_tags__contains`, which will most likely be the way it's added on backend (see https://github.com/openfoodfacts/open-prices/issues/673)
* Some mobile friendly UI tweaks

New page look:
![Screenshot 2025-01-11 at 17-28-46 Open Prices](https://github.com/user-attachments/assets/0b1e28e9-5e12-4ff6-a913-7086a9604b5d)

raphodn reviewed (approved):
lets go !
